common/app/model/content/Atoms.scala

```diff
@@ -127,6 +135,21 @@ object MediaAtom extends common.Logging {
       mimeType = mediaAsset.mimeType)
   }
 
+  def mediaImageAssetMake(mediaImage: AtomApiImageAsset, caption: String): ImageAsset = {
+    ImageAsset(
+      index = 0,
```

Comment:
@akash1810 I believe this is used to indicate the index of images when used in a gallery context. I am going to default it to `0` in the case class, as it is set to that in most instances.
CC @NataliaLKB